src/qt/receivecoinsdialog.cpp

```diff
@@ -46,6 +46,9 @@ ReceiveCoinsDialog::ReceiveCoinsDialog(const PlatformStyle *_platformStyle, QWid
         ui->removeRequestButton->setIcon(QIcon(":/icons/remove"));
     }
 
+    ui->reqLabel->setPlaceholderText(tr("Enter a label for this request to add its address to your address book"));
```

Comment:
The address is going to be added to the address book regardless, even with an empty label. Should be
```suggestion
    ui->reqLabel->setPlaceholderText(tr("Enter a label the address generated for this request should have in your address book"));
```
or smth like that imo

Reply:
What about de54493 ?

Reply:
Hmm.. How about reusing a wording similar to the one in the corresponding tooltip https://github.com/dashpay/dash/blob/master/src/qt/forms/receivecoinsdialog.ui#L67? e.g. `Enter a label to associate with the new receiving address` (and the same idea for the message https://github.com/dashpay/dash/blob/master/src/qt/forms/receivecoinsdialog.ui#L74 i.e. `Enter a message to attach to the payment request`).

As a side note, we should probably add placeholders via `placeholderText` property in forms, not via functions in code.

Reply:
> Hmm.. How about reusing a wording similar to the one in the corresponding tooltip https://github.com/dashpay/dash/blob/master/src/qt/forms/receivecoinsdialog.ui#L67? e.g. `Enter a label to associate with the new receiving address` (and the same idea for the message https://github.com/dashpay/dash/blob/master/src/qt/forms/receivecoinsdialog.ui#L74 i.e. `Enter a message to attach to the payment request`).

That sounds like a good idea :) 

> As a side note, we should probably add placeholders via `placeholderText` property in forms, not via functions in code.

Hmm.. there are already other set in code, whats the reason you think about here?

Reply:
re placeholders: https://github.com/bitcoin/bitcoin/pull/17702

Reply:
> Hmm.. How about reusing a wording similar to the one in the corresponding tooltip https://github.com/dashpay/dash/blob/master/src/qt/forms/receivecoinsdialog.ui#L67? e.g. `Enter a label to associate with the new receiving address` (and the same idea for the message https://github.com/dashpay/dash/blob/master/src/qt/forms/receivecoinsdialog.ui#L74 i.e. `Enter a message to attach to the payment request`).

Done in 745075c